src/main/java/com/gradle/exportapi/dao/BuildDAO.java

```diff
@@ -21,8 +24,15 @@
     public static long insertBuild(Build build) {
         LOGGER.debug("Inserting build {} into the database.", build.getBuildId());
 
-        OffsetDateTime start = OffsetDateTime.ofInstant(build.getTimer().getStartTime(), ZoneId.of(build.getTimer().getTimeZoneId()));
-        OffsetDateTime finish = OffsetDateTime.ofInstant(build.getTimer().getFinishTime(), ZoneId.of(build.getTimer().getTimeZoneId()));
+        Object start;
+        Object finish;
+        if(SqlHelper.isMySql()) {
+            start = LocalDateTime.ofInstant(build.getTimer().getStartTime(), ZoneOffset.UTC);
```

Comment:
filed https://github.com/gradle/ge-export/issues/18. you can add it to the comment